src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -375,3 +375,30 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Current Culture Context
+
+Current Culture Context describes certain properties of the culture in which 
+the software is used.
+
+The `type` and default key is `"current_culture"`.
```

Comment:
Please use `type: culture` here, since that is the true type of the context. You can still have a `current_culture` or other key for SDK-specific conventions.